src/Data/Ord.purs

```diff
@@ -214,10 +214,12 @@ between low hi x
 abs :: forall a. Ord a => Ring a => a -> a
 abs x = if x >= zero then x else negate x
 
--- | The sign function; always evaluates to either `one` or `negate one`. For
--- | any `x`, we should have `signum x * abs x == x`.
+-- | The sign function; always evaluates to `negate one`, `zero` or `one`.
```

Comment:
This comment doesn't really tell you what the function does. Also I think the second sentence can stay - that should still be true, surely?

Reply:
```suggestion
-- | The sign function; returns `one` if the argument is positive, `negate one` if the argument is negative, or `zero` if the argument is `zero`. For any `x`, we should have `signum x * abs x == x`.
```